infrastructure/src/main/java/org/corfudb/infrastructure/ServerContext.java

```diff
@@ -126,4 +129,17 @@ public long getStartingAddress() {
     public void setStartingAddress(long startingAddress) {
         dataStore.put(Long.class, PREFIX_STARTING_ADDRESS, KEY_STARTING_ADDRESS, startingAddress);
     }
+
+    /**
+     * The ID for this cluster.
+     */
+    public synchronized UUID getClusterId() {
+        UUID clusterId = dataStore.get(UUID.class, PREFIX_CLUSTER_ID, KEY_CLUSTER_ID);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "clusterId". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)